tests/backup/backup_helper.go

```diff
@@ -5559,3 +5561,172 @@ func UpgradeKubevirt(versionToUpgrade string, workloadUpgrade bool) error {
 	}
 	return nil
 }
+
+// UpdateKDMPConfigMap update the KDMP configMap with give data key and value.
```

Comment:
nit: `UpdateKDMPConfigMap updates the KDMP configMap with the given key and value.`